content/about.md

```diff
@@ -3,7 +3,32 @@ title: "About"
 date: "2021"
 ---
 
-### External links
+Or is an experienced SRE for over a decade, building companies and their infrastructure.
+Responsible for educating engineers working with cloud-native platforms on best-practices,
```

Comment:
Establishing and leading on-call culture based on ownership, innovation while utilizing most cutting-edge tools and infrastructures.